Duckling/Time/EN/Corpus.hs

```diff
@@ -163,6 +163,14 @@ latentCorpus = (testContext, testOptions {withLatent = True}, xs)
              ,  "twelve ou three"
              ,  "twelve oh three"
              ]
+      , examples (datetime (2013, 2, 12, 4, 30, 0) Second)
+             [ "now"
+             , "right now"
+             , "just now"
+             ]
+      , examples (datetime (2013, 2, 15, 0, 0, 0) Day)
+             [ "on the 15th"
+             ]
```

Comment:
The reason to make these corpus latent is that these datetime may not only belong to time dim? Not sure if I understand this correct?

Reply:
Yeah, that's correct. This is one of the time patterns which are too ambiguous to treat as non-latent without more context